superset/examples/birth_names.py

```diff
@@ -41,6 +41,12 @@
 )
 
 admin = security_manager.find_user("admin")
+if admin is None:
+    raise Exception(
```

Comment:
Let's raise one of these https://github.com/apache/incubator-superset/blob/master/superset/exceptions.py#L55

Reply:
Great I will. Thanks for suggestion :)

Reply:
I've chosen `NoDataException`